openfga_sdk/sync/client/client.py

```diff
@@ -0,0 +1,675 @@
+# coding: utf-8
```

Comment:
Let's make sure this top level `client` has an `__init__.py` in it.

Reply:
Will do